src/StatementParsers/TransactionParser.php

```diff
@@ -33,19 +34,33 @@ public function parse(array $lines): Transaction
 		$valutaDate = new DateTime("0001-01-01");
 		$amount = 0.0;
 		$sepaDirectDebit = null;
+
+        /** @var string|null $transactionSequence */
+        $statementSequence = null;
+
+		/** @var string|null $transactionSequence */
+        $transactionSequence = null;
```

Comment:
Type of the sequence numbers should be "int".
Fields should not be nullable and be "0" by default (also see values above).